debian/yaru-theme-gtk.install

```diff
@@ -1,7 +1,13 @@
 usr/share/gtksourceview-*/styles/Yaru*.xml
```

Comment:
@Muqtxdir I think you forgotten gtksourceview in your previous commit ;)

Reply:
ok, thanks I'll change it 👍